Directory.Build.props

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <!-- support building in non msbuild 15 -->
+    <DirectoryBuildPropsImported>true</DirectoryBuildPropsImported>
+    <EnlistmentRoot>$(MSBuildThisFileDirectory.TrimEnd('\\'))</EnlistmentRoot>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+
+    <BaseIntermediateOutputPath Condition=" '$(BaseIntermediateOutputPath)'=='' ">$(EnlistmentRoot)\obj\</BaseIntermediateOutputPath>
+    <IntermediateOutputPath Condition=" '$(IntermediateOutputPath)'=='' ">$(BaseIntermediateOutputPath)$(Configuration)\$(Platform)\$(MSBuildProjectFile)</IntermediateOutputPath>
+    <RestoreOutputPath>$(IntermediateOutputPath)</RestoreOutputPath>
+    <OutputPath Condition=" '$(OutputPath)' == '' ">$(EnlistmentRoot)\bin\$(Configuration)\$(Platform)\$(MSBuildProjectName)</OutputPath>
+    <TargetFrameworkVersion Condition=" '$(TargetFrameworkVersion)' == '' ">v4.5</TargetFrameworkVersion>
+    <!-- Temporarily pin to version 3.4 -->
+    <CBTNuGetDownloaderArguments>https://dist.nuget.org/win-x86-commandline/v3.4.4/NuGet.exe</CBTNuGetDownloaderArguments>
```

Comment:
This is needed until the cbt.nuget, cbt.nuget.globalpackages, cbt.traversal, cbt.unifiedoutput modules are updated to work with 4.x  since the cbt.nuget module uses this downloaded version of nuget.  Technically for this branch just updating cbt.nuget will be sufficient.